lib/actions/userActions/index.js

```diff
@@ -0,0 +1,26 @@
+const i18n               = require('../../../config/i18n')
+const ch                 = require('../../chatHelpers')
+const allActions         = require('../')
+
+const actions = {
+  // Used to request most recent geo location of user
+  'user:get_geo': (payload) => {
+    return ch.respondWith({
+      text: ``,
+      responseType: `userLocation`,
```

Comment:
@roblafeve check for a `responseType` of `userLocation`, get the updated location of the user, and handle the rest like a normal response.